README.md

```diff
@@ -1,14 +1,103 @@
+| ```SPT```         | Compton-y map                     | [Bleem et al, 2021](https://arxiv.org/abs/2102.05033)                 | [catalogue](https://lambda.gsfc.nasa.gov/product/spt/spt_prod_table.html)                         | J. Ruiz-Zapatero       |
+| ```WIxSC```       | Galaxy clustering                 | [Bilicki et al, 2016](https://arxiv.org/abs/1607.01182)               | [catalogue](http://ssa.roe.ac.uk/WISExSCOS.html)                                                  | D. Alonso              |
 
---- 
-More info about the sacc files in https://github.com/LSSTDESC/sacc
+# Design Philosophy
+![](https://raw.githubusercontent.com/JaimeRZP/Cosmoteka_tutorials/master/docs/src/assets/Cosmoteka_schematic_v2.png)
+
+Cosmoteka is a large collection of pipelines to process map-level data from a wide variaty of surveys into angular power spectra combined in a statistically consistent manner. Cosmoteka accomplishes this by computing all the cross- and auto-covariances of the different maps.
+
+Cosmoteka is designed to allow for the largest amount of modularity possible to encourage open-source community development. Inside of each module, Cosmoteka follows an object-oriented approach. Thus, given some configureation file, each module defines a series of objects which host the methods used to process the data at each step of the pipeline. 
+
+Cosmoteka is fundamentally divided in two modules, `cls` and `mappers`, which can be thought as the brain and muscles respecively of the same organism. `cls` processes the demands the user can make through a configuration file and reaches out for the relevant `mapper` modules to compute the  corresponding `NaMaster` fields. In order to understand, how the `mapper` modules compute the `NaMaster` fields we can look the `mapper_base.py`, the mapper module which defines the parent class for all other mappers inside Cosmoteka. As the parent class, `mapper_base` defines a series of default methods which are then overwritten by the child mappers to perform the data processing specific to each data set. Moreover, it also defines common methods to each mapper. The most important of these methods is `get_nmt_field` which returns the `NaMaster` field from each mapper. Inside `get_nmt_field`, the signal map and mask of each mapper are computed according to the specifc methods of the children mappers. Then, the signal map and mask as well as other mapper specific quantities (data beam, contaminants,... etc) are passed on to `NaMaster` to compute the final field. Once the `NaMaster` fields are computed, `cls` orchestrates the computation of the angular power spectra and their covariance requested by the user. A brief description of the role of each module inside `cls` can be found in the table below. For a more detailed description of the role of the  `cls` modules as well as the individual mappers please visit the documentation of Cosmoteka.
```

Comment:
I find this a bit confusing and too long. I would just let the table below talk by itself. Add  `mappers` there and explain what you say here: they project the catalogs into sky maps and provide `NmtField`s.